### ISSUE TITLE
Allow footer compatible with sidebar

### ISSUE THREAD

**Issue by Otto-J** [vuejs/vitepress#1037]
### Is your feature request related to a problem? Please describe.

Now `footer` and `sidebar` cannot be used at the same time. We hope they can appear at the same time. 

### Describe the solution you'd like

- footer and sidebar can used at the same time

### Describe alternatives you've considered

If allowed, maybe I can provide a PR？

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Read the [Contributing Guidelines](https://github.com/vuejs/vitepress/blob/main/.github/contributing.md).
- [X] Check that there isn't already an issue that asks for the same feature to avoid creating a duplicate.

**Comment by Otto-J** [comment]
oh same https://github.com/vuejs/vitepress/pull/1034#issue-1313570701

**Comment by Otto-J** [comment]
anyway, thank you for your attention

**Comment by brc-dd** [comment]
It doesn't look nice with sidebar, that's why its not there. If you want to create a PR, then make it optional to enable footer (using frontmatter or some theme config) on doc pages.

**Comment by xsjcTony** [comment]
Yeah honestly it would be good, especially if we need to display some must-have copyright info or something else.

**Comment by rocex** [comment]
Here's how I solved it
in .vitepress\theme\style.css add this

```css
footer.VPFooter {
    display: block !important;
}

```

**Comment by ronny1020** [comment]
Instead of enforcing a specific design choice, wouldn't it be better to let users decide whether to use the footer, the sidebar, or both? If someone doesn't like the look, they could simply remove or hide the footer/sidebar in their own configuration. Is there a specific technical reason why this flexibility isn't allowed?
